itest/rust/src/object_tests/dyn_gd_test.rs

```diff
@@ -507,23 +507,35 @@ impl InstanceIdProvider for foreign::NodeHealth {
 }
 
 // ----------------------------------------------------------------------------------------------------------------------------------------------
-// Check if DynGd can be properly exported
+// Checks if DynGd can be properly used as a `#[var]`.
 
 #[derive(GodotClass)]
 #[class(init)]
-struct RefcDynGdExporter {
+struct RefcHasDynGdVar {
```

Comment:
```suggestion
struct RefcDynGdVarDeclarer {
```
maybe? :sweat_smile: